test/unit/quality-labels-test.js

```diff
@@ -1,10 +1,6 @@
-import _ from 'utils/underscore';
```

Comment:
Great suggestion Rob! I merged all the sub-function unit tests so that they are tested through `generateLabel` instead and no longer need to be exported.

Reply:
Thanks!